screenshot_to_code/backend/config.py

```
@@ -6,12 +6,12 @@
 NUM_VARIANTS = 2
 
 # LLM-related
-OPENAI_API_KEY = os.environ.get("OPENAI_API_KEY", None)
-ANTHROPIC_API_KEY = os.environ.get("ANTHROPIC_API_KEY", None)
+OPENAI_API_KEY = os.environ.get("OPENAI_API_KEY", "")
```

Empty string default values for API keys could lead to silent failures. Consider
raising an error when required API keys are missing.

---

/implement